src/components/NotificationsDrawer/DrawerPanelContent.tsx

```diff
@@ -124,7 +136,23 @@ const DrawerPanelBase = ({ innerRef }: DrawerPanelProps) => {
         );
       }
     };
+    const fetchFilterConfig = async () => {
```

Comment:
Why is this function defined in `useEffect`?

Reply:
I was following how the permissions were fetched after `mounted` became true in the `fetchPermissions` method.

Reply:
I see. The pattern used before is wrong. The mounted variable should be in ref so we don't have to do this. Its something for a future refactoring I guess.